specification/resource/semantic_conventions/README.md

```diff
@@ -76,6 +77,7 @@ as specified in the [Resource SDK specification](https://github.com/open-telemet
 | Attribute  | Type | Description  | Examples  | Requirement Level |
 |---|---|---|---|---|
 | `service.name` | string | Logical name of the service. [1] | `shoppingcart` | Required |
+| `service.version` | string | The version string of the service API or implementation. | `2.0.0` | Recommended |
```

Comment:
We could consider adding a second example so readers don't mistakenly assume it's always a semver string.

```suggestion
| `service.version` | string | The version string of the service API or implementation. | `2.0.0`, `a01dbef8a` | Recommended |
```